Main.dart

```diff
@@ -0,0 +1,40 @@
+void main()
+{
+    Pessoa p1 = Pessoa("Pedro", 1.76, DateTime.parse("2003-06-12"));
+    print(p1.dados());
+
+}
+
+class Pessoa {
+
+    String _nome;
+    double _altura;
+    DateTime _niver;
+
+    Pessoa(String nome, double altura, DateTime niver) {
```

Comment:
no dart tem uma coisa legal de atribuir à classe no construtor, procura por "this." no construtor do dart :-)